docs/migration-to-NRI.md

```diff
@@ -0,0 +1,182 @@
+# Prerequisities
+- Up and running CRI Resource Manager
+- One of the two supported policies in use: balloons or topology-aware.
+- For other policies a little bit more work is required and the policies need to be 'ported'. This can be done by just following the example of how balloons or topology-aware was converted.
+
+# Steps for an initial/basic migration test
+## Containerd 
+- Replace the containerd version in the system with 1.7 or newer version (NRI server not supported in older versions).
```

Comment:
nit: I think the bullet list looks a bit funny and doesn't really bring any value (as the section does not have any other content than bullets). I suggest to not use bullets

```suggestion

Replace the containerd version in the system with 1.7 or newer version (NRI server not supported in older versions).
```

Also prolly applies to some other sections here